lib/util.js

```diff
@@ -433,6 +443,22 @@ function formatValue(ctx, value, recurseTimes, ln) {
     return ctx.stylize('null', 'null');
   }
 
+  if (getClockTime(ctx.startTime) > ctx.maxClockTime) {
```

Comment:
Cache the result of `getClockTime()`.  `process.hrtime()` is not necessarily cheap.